### DISCUSSION TITLE
Template for child of x note

### DISCUSSION THREAD

**Discussion by shirobachi** [TriliumNext/Trilium#3261] [Q&A] [answered]
Hey, 

I have note called recipes, and a template (somewhere else in tree) called recipe. My goal is to paste recipe content is new note created under recipes note.
I know the relation `~template` but does not work for me, I think. It only pastes recipe content into recipes note - it's not what I want.
How can I do this?

I had a look for similar discussion posts and issues in repo. Also had a look for wiki with Google little help but did not find the answer. 
I am new in Trilium, so maybe I am missing some great resource for new member?

Thank you in advance!

**Comment by shirobachi** [comment, marked as answer]
I found this relation in demo `~child:template=@page` what should work. I just believe that maybe wiki is not clear enough (or I missed it). What would you guys say to add this to wiki/template, I think it's worth to do this.

**Reply by shirobachi** [reply]
Oh, and `~child:template` is not suggested instead of `~template` I mean this as suggested: https://i.imgur.com/cSqNNQ3.png

**Comment by davidszerman** [comment]
Very useful, thank you ! Thanks to you I understood how to use the templates  I am creating
